argilla/docs/tutorials/image_classification.ipynb

```diff
@@ -0,0 +1,804 @@
+{
```

Comment:
<p>Why do we need to do this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>@burtenshaw, it is stated above "Specifically, we will use <code>100</code> examples. Because we are dealing with a potentially large image dataset, we will set <code>streaming=True</code> to avoid loading the entire dataset into memory and iterate over the data to lazily load it."</p>

Reply:
@burtenshaw, it is stated above "Specifically, we will use 100 examples. Because we are dealing with a potentially large image dataset, we will set streaming=True to avoid loading the entire dataset into memory and iterate over the data to lazily load it."